From 4443476e05ce9905c18fa6fb7591795de977e444 Mon Sep 17 00:00:00 2001
From: Bastian Zimmermann <10774221+BastianZim@users.noreply.github.com>
Date: Thu, 16 Sep 2021 18:41:04 +0200
Subject: [PATCH] Patch for downstream

---
 CoinUtils/coinutils.pc.in | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/CoinUtils/coinutils.pc.in b/CoinUtils/coinutils.pc.in
index 235ffaf6..05b6fbb2 100644
--- a/CoinUtils/coinutils.pc.in
+++ b/CoinUtils/coinutils.pc.in
@@ -7,6 +7,7 @@ Name: CoinUtils
 Description: COIN-OR Utilities
 URL: https://projects.coin-or.org/CoinUtils
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lCoinUtils @COINUTILSLIB_PCLIBS@
+Libs: -L${libdir} -lCoinUtils 
+Libs.private: @COINUTILSLIB_PCLIBS@
 Cflags: -I${includedir}
-Requires: @COINUTILSLIB_PCREQUIRES@
+Requires.private: @COINUTILSLIB_PCREQUIRES@
-- 
2.30.1 (Apple Git-130)

